controllers/config_test.go

```diff
@@ -181,3 +182,89 @@ func TestGetProviderControllerFromImages(t *testing.T) {
 		})
 	}
 }
+
+func TestComposeConfig(t *testing.T) {
```

Comment:
Can we use ginkgo for this test?

Reply:
I don't think it is needed. We use ginkgo because of the comfort working with real API server in C-R, and this is just a unit test for method which does no API requests.

Reply:
Could use gomega still, definitely prefer that personally, rather than `t.Fatal`, but up to you :)

Reply:
https://github.com/openshift/cluster-cloud-controller-manager-operator/pull/41 is refactoring this in preparation for bootstrap support in https://github.com/openshift/cluster-cloud-controller-manager-operator/pull/42. It is using `assert` which is very convenient. I still think gomega should be used in cases where there is a need to operator with API server, and this is not the case. `DescribeTable` is very unintuitive, and with use of assert simplified this tests enough that AfterEach and BeforeEach does not add value.